# nearest points between objects
# pointのstampのframe_idが、リンクを表す
geometry_msgs/PointStamped point1
geometry_msgs/PointStamped point2

geometry_msgs/Vector3Stamped direction21

float64 distance
